sql/mysql/migrate.go

```diff
@@ -377,7 +377,11 @@ func (s *state) alterTable(t *schema.Table, changes []schema.Change) error {
 			reverse.Comma().P("DROP FOREIGN KEY").Ident(change.F.Symbol)
 		case *schema.DropForeignKey:
 			b.P("DROP FOREIGN KEY").Ident(change.F.Symbol)
-			reversible = false
+			reverse.Comma().P("ADD")
```

Comment:
A nice approach to reduce duplications (also in drop index), is to:

reverse.Comma()
b, reverse = reverse, b
defer func() { b, reverse = reverse, b }()
falltrough to add version.

Reply:
`fallthrough` does not work in type switch

Reply:
right :| 

kk, I have another idea, but let's merge this one atm